### ISSUE TITLE
For a two-key shortcut both keys have to be pressed each time

### ISSUE THREAD

**Issue by mardukbp** [nhmkdev/KeyCap#5]
I defined the mapping Shift+Backspace = Delete. The first time I have to press both keys and it works. If I leave Shift pressed and then press Backspace the key event sent is Backspace instead of Delete. KeyCap should check if a key is already pressed whenever a key is pressed and then both together would match the defined shortcut.

Thanks for writing this awesome program!

**Comment by nhmkdev** [comment]
Which program are you attempting to use Shift+Backspace in? I noticed in notepad it works as desired (you can hold down shift and hit backspace numerous times without letting go of shift). In TextPad it did not work (at all, ha!).

**Comment by mardukbp** [comment]
For me in Notepad it does not work as desired. I configured the shortcut like this:
![grafik](https://user-images.githubusercontent.com/1140106/149473454-1690f83c-cc40-48c3-a9fc-504ccfe529eb.png)

In VS Code and Outlook I have to press both keys each time.

I downloaded the latest release, 2.0.0.1. But the window title says 2.0.0.2.

**Comment by nhmkdev** [comment]
Haha, well I guess fixing the version number in the next revision would be nice...

I'll upload a new version to try out. My settings appear to be about the same. Every application I've tried seems to react differently to shift+backspace (potentially having hooks of their own to perform a different action). Very weird...

**Comment by nhmkdev** [comment]
Here's something to check: What does left shift vs. right shift do for you? In notepad left-shit + backspace acts as you described (working once) but right-shift works numerous times.

**Comment by mardukbp** [comment]
You are right. Right shift + Backspace works in Notepad, but in VS Code it seems to be hard wired to the Cut command. So in KeyCap I mapped Ctrl + Alt + Backspace to Delete and that does not work continuously. I implemented that mapping with AHK and it works. Given that it is open source, maybe you can look in the source code to see how they handle it.
